feature/detail/src/main/java/team/ppac/detail/component/DetailContent.kt

```diff
@@ -194,15 +194,20 @@ internal fun DetailHashTags(
         modifier = Modifier.mapTextSkeletonModifierIfNeed(isLoading = isLoading, height = 18.dp),
         hashTags = hashTags.truncateDisplayedList(6),
     )
-    Spacer(modifier = Modifier.height(11.dp))
-    Text(
-        modifier = Modifier.mapTextSkeletonModifierIfNeed(isLoading = isLoading, height = 15.dp),
-        text = sourceDescription.truncateDisplayedString(27),
-        color = FarmemeTheme.textColor.assistive,
-        style = FarmemeTheme.typography.body.xSmall.medium,
-        maxLines = 1,
-        overflow = TextOverflow.Ellipsis
-    )
+    sourceDescription.takeIf { it.isNotEmpty() }?.let {
+        Spacer(modifier = Modifier.height(11.dp))
+        Text(
+            modifier = Modifier.mapTextSkeletonModifierIfNeed(
+                isLoading = isLoading,
+                height = 15.dp,
+            ),
+            text = "출처 : ${it.truncateDisplayedString(32)}",
+            color = FarmemeTheme.textColor.assistive,
+            style = FarmemeTheme.typography.body.xSmall.medium,
+            maxLines = 1,
+            overflow = TextOverflow.Ellipsis,
+        )
+    }
```

Comment:
NIT : 요런식으로 간단하게 갈 수도 있을 것 같아서 제안해봤습니당 ㅋㅋ
```suggestion
    if (sourceDescription.isNotEmpty()) {
        Spacer(modifier = Modifier.height(11.dp))
        Text(
            modifier = Modifier.mapTextSkeletonModifierIfNeed(
                isLoading = isLoading,
                height = 15.dp,
            ),
            text = "출처 : ${it.truncateDisplayedString(32)}",
            color = FarmemeTheme.textColor.assistive,
            style = FarmemeTheme.typography.body.xSmall.medium,
            maxLines = 1,
            overflow = TextOverflow.Ellipsis,
        )
    }
```

Reply:
@JaesungLeee  수정완룡!!